### PR TITLE
[i18n-ar] add README_ar.md to README.md

### PR DESCRIPTION
**Pull request by AhmedAlmaghz**
# What does this PR do? 
 Translate docs into Arabic issue-number #32435 
 @stevhliu , @MKhalusova , @abodacs 
 This pull request contains changes [**for file : README.md**](https://github.com/AhmedAlmaghz/transformers/blob/Update_README_md/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ limitations under the License.
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_fr.md">Français</a> |
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_de.md">Deutsch</a> |
         <a href="https://github.com/huggingface/transformers/blob/main/i18n/README_vi.md">Tiếng Việt</a> |
+		<a href="https://github.com/huggingface/transformers/blob/main/i18n/README_ar.md">العربية</a> |
```
Comment (abodacs) [inline README.md:51]: @AhmedAlmaghz 
Please keep the same indent .

**Review by abodacs — CHANGES_REQUESTED** [review]
@AhmedAlmaghz 
Can you change the title to reflect the commit content?

I recommend renaming it to  "add README_ar.md to README.md"

**Comment by abodacs** [comment]
LGTM 

Thanks @AhmedAlmaghz

File: i18n/README_ar.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+يسمح السطر الثاني من التعليمات البرمجية بتحميل النموذج المُدرب مسبقًا الذي يستخدمه خط الأنابيب وتخزينه مؤقتًا، بينما يقوم السطر الثالث بتقييمه على النص المحدد. هنا، تكون الإجابة "إيجابية" بثقة تبلغ 99.97%.
+
+تتوفر العديد من المهام على خط أنابيب مُدرب مسبقًا جاهز للاستخدام، في NLP ولكن أيضًا في رؤية الكمبيوتر والخطاب. على سبيل المثال، يمكننا بسهولة استخراج الأشياء المكتشفة في صورة:
+
+``` python
+>>> import requests
+>>> from PIL import Image
+>>> from transformers import pipeline
+
+# قم بتنزيل صورة بها قطط لطيفة
```
Comment (stevhliu) [inline i18n/README_ar.md:154]: Nice, thanks for translating the code comments too!

**Review by stevhliu — COMMENTED** [review]
Thanks! I would double-check the Markdown links are correctly formatted. Some of the bracketed items are separated from the parentheses kind of like below:

```md
[some text] more text more text (link)
```

File: i18n/README_ar.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+
+يمكن تطبيق هذه النماذج على:
+
+* 📝 النص، لمهام مثل تصنيف النص واستخراج المعلومات والرد على الأسئلة والتلخيص والترجمة وتوليد النص، في أكثر من 100 لغة.
+* 🖼️ الصور، لمهام مثل تصنيف الصور وكشف الأشياء والتجزئة.
+* 🗣️ الصوت، لمهام مثل التعرف على الكلام وتصنيف الصوت.
+
+يمكن لنماذج المحول أيضًا أداء مهام على **طرائق متعددة مجتمعة**، مثل الرد على الأسئلة الجدولية والتعرف البصري على الحروف واستخراج المعلومات من المستندات الممسوحة ضوئيًا وتصنيف الفيديو والرد على الأسئلة المرئية.
+
+يوفر 🤗 Transformers واجهات برمجة التطبيقات (APIs) لتحميل تلك النماذج المُدربة مسبقًا واستخدامها على نص معين، وضبطها بدقة على مجموعات البيانات الخاصة بك، ثم مشاركتها مع المجتمع على [مركز النماذج](https://huggingface.co/models) الخاص بنا. وفي الوقت نفسه، فإن كل وحدة نمطية Python التي تحدد بنية هي وحدة مستقلة تمامًا ويمكن تعديلها لتمكين تجارب البحث السريعة.
```
[the remark this answers is not in the document]
Comment (DSYemen) [reply i18n/README_ar.md:73]: Yes!

[the remark this answers is not in the document]
Comment (stevhliu) [reply i18n/README_ar.md:73]: Great, thanks!

**Review by stevhliu — APPROVED** [review]
Thanks for translating to Arabic, and thanks to all the reviewers as well 🤗

**Comment by AhmedAlmaghz** [comment]
Thank you @stevhliu  on merged

**Pull request merged**